ts/state/reducer.ts

```diff
@@ -27,9 +33,15 @@ export type StateType = {
   userConfig: UserConfigState;
 };
 
+const conversationsPersistConfig = {
+  key: 'conversations',
+  storage,
+  whitelist: ['conversationLookup'],
+};
+
 export const reducers = {
   search,
-  conversations,
+  conversations: persistReducer(conversationsPersistConfig, conversations),
```

Comment:
better to not make that persist on redux. if we ever want to move conversation attributes to redux we will move them all in a single commit.
We already have a copy on them in the database, a memory copy in the conversation controller. Better to not make the app more complicated for now